content/concepts/consensus-network/amendments/known-amendments.md

```diff
@@ -3,18 +3,19 @@
 
 The following is a comprehensive list of all known amendments and their status on the production XRP Ledger:
 
-| Name                            | Introduced | Status                              |
-|:--------------------------------|:-----------|:------------------------------------|
+| Name                            | Introduced | Status                        |
+|:--------------------------------|:-----------|:------------------------------|
+| [fix1781][]                     | v1.6.0-b1  | [In Development: TBD]( "BADGE_LIGHTGREY") |
 | [CryptoConditionsSuite][]       | TBD        | [In Development: TBD]( "BADGE_LIGHTGREY") |
```

Comment:
This is no longer going to be published right? @seelabs  Do we want to do something about this line?

Reply:
What's no longer going to be published?

Reply:
The project may be resurrected at some point, but there are no short term plans to merge cryptoconditions. Up to you if you want to remove the line.

Reply:
I'll wait until there's a `rippled` beta that removes the amendment.